java/src/main/java/com/google/cloud/dataproc/templates/hbase/HbaseToGCS.java

```diff
@@ -61,20 +61,26 @@ public void runTemplate() {
     }
 
     SparkSession spark = SparkSession.builder().appName("Spark HbaseToGCS Job").getOrCreate();
-
-    Map<String, String> optionsMap = new HashMap<>();
-    optionsMap.put(HBaseTableCatalog.tableCatalog(), catalogue.replaceAll("\\s", ""));
-
-    // Read from HBase
-    Dataset dataset =
-        spark
-            .read()
-            .format("org.apache.hadoop.hbase.spark")
-            .options(optionsMap)
-            .option("hbase.spark.use.hbasecontext", "false")
-            .load();
-
-    // Write To GCS
-    dataset.write().format(outputFileFormat).mode(gcsSaveMode).save(gcsWritePath);
+    Dataset dataset = null;
+    try {
+      Map<String, String> optionsMap = new HashMap<>();
+      optionsMap.put(HBaseTableCatalog.tableCatalog(), catalogue.replaceAll("\\s", ""));
+      // Read from HBase
+      dataset =
+          spark
+              .read()
+              .format("org.apache.hadoop.hbase.spark")
+              .options(optionsMap)
+              .option("hbase.spark.use.hbasecontext", "false")
+              .load();
+    } catch (NoClassDefFoundError error) {
```

Comment:
Adding try catch just for "test case" is not an option.
We don't need to junit test spark session and onwards code.
Look at HiveToGCSTest.java to create test accordingly.